project-addons/crm_claim_rma_custom/crm_claim.py

```diff
@@ -34,10 +35,23 @@ class equivalent_products_wizard(models.TransientModel):
 
 
 class CrmClaimRma(models.Model):
-
     _inherit = "crm.claim"
     _order = "id desc"
 
+    @api.one
+    def _has_category(self, expected_category):
+        has_category = False
+        for category in self.category_id:
+            if category.parent_id.id == expected_category.id:
+                has_category = True
+        self.bool_category_id = has_category
+
+    @api.multi
+    def _check_category_id(self):
```

Comment:
Ok, como veáis, pero hacéis compute="_has_category" y le elimináis el parámetro extra, substituyéndolo por una variable interna y ya casi estaría.